homeassistant/components/epion/strings.json

```diff
@@ -0,0 +1,32 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Define the API key (token) for this installation",
```

Comment:
This sounds more like a description. It doesn't add a lot in my opinion. You can use `data_description` fields to add more context to the api_key field if needed, otherwise i'd say remove it